cmd/operator-sdk/run/cmd.go

```diff
@@ -34,6 +34,8 @@ import (
 	hoflags "github.com/operator-framework/operator-sdk/pkg/helm/flags"
 )
 
+// todo(camilamacedo86): the command run --local still required for Ansible/Helm projects using the new layout
+// It was affected by the PR #3253. In this way, it needs to be added here.
```

Comment:
Remove before the merge. it is done here: https://github.com/operator-framework/operator-sdk/pull/3330
```suggestion
```